internal/build_defs.bzl

```diff
@@ -132,33 +125,35 @@ def _ts_library_impl(ctx):
 ts_library = rule(
     _ts_library_impl,
     attrs = dict(COMMON_ATTRIBUTES, **{
-        "srcs":
-            attr.label_list(
-                allow_files=FileType([
-                    ".ts",
-                    ".tsx",
-                ]),
-                mandatory=True,),
+        "srcs": attr.label_list(
+            allow_files = FileType([
+                ".ts",
+                ".tsx",
+            ]),
+            mandatory = True,
+        ),
 
         # TODO(alexeagle): reconcile with google3: ts_library rules should
         # be portable across internal/external, so we need this attribute
         # internally as well.
-        "tsconfig":
-            attr.label(allow_files = True, single_file = True),
-        "compiler":
-            attr.label(
-                default=get_tsc(),
-                single_file=False,
-                allow_files=True,
-                executable=True,
-                cfg="host"),
+        "tsconfig": attr.label(
+            allow_files = True,
```

Comment:
curious, did you run buildifier? we should make that part of the process (maybe in a git pre-commit hook?)